scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/TestRecurrenceGoal.java

```diff
@@ -48,4 +49,32 @@ public void testRecurrence() {
     assertTrue(TestUtility.activityStartingAtTime(plan,Duration.of(16, Duration.SECONDS), activityType));
   }
 
+  @Test
+  public void testRecurrenceNegative() {
+    var planningHorizon = new PlanningHorizon(TestUtility.timeFromEpochSeconds(0),TestUtility.timeFromEpochSeconds(20));
+    final var fooMissionModel = SimulationUtility.getFooMissionModel();
+    Problem problem = new Problem(fooMissionModel, planningHorizon, new SimulationFacade(
```

Comment:
:large_orange_diamond: There's some small formatting tweaks to be done in this method. E.g. You can align the args to `Problem` one each on a new line. Drop the second `Duration` arg in `.forAllTimeIn` to a new line to reduce the line length. Use a text block for the message in your assertion.

Reply:
I suspect all of the variables on the highlighted lines can be made `final var`, too, as well as the `RecurrenceGoal` some lines down.